### PR TITLE
BZ1926248: Manual changes for upgrading MTC 1.3 to 1.4 (1.4.0)

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1926248

Urgent. MTC 1.4.0 release is Feb. 10.

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

**Comment by XinRedhat** [comment]
LGTM

File: modules/migration-upgrading-from-mtc-1-3.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+----
+$ oc replace -f <migplan>.yaml -n openshift-migration
+----
+
+. Verify the changes by viewing the updated `MigPlan` CR:
++
+[source,terminal]
+----
+$ oc get migplan <migplan> -o yaml -n openshift-migration
+----
```
Comment (kalexand-rh) [inline modules/migration-upgrading-from-mtc-1-3.adoc:50]: If you have output handy, I'd include the sample output.

**Review by kalexand-rh — COMMENTED** [review]
This change is looking good!

File: modules/migration-upgrading-from-mtc-1-3.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// * migration/migrating_3_4/upgrading-migration-tool-3-4.adoc
+// * migration/migrating_4_1_4/upgrading-migration-tool-4-1-4.adoc
+// * migration/migrating_4_2_4/upgrading-migration-tool-4-2-4.adoc
+
+[id="migration-upgrading-from-mtc-1-3_{context}"]
+= Upgrading {mtc-short} 1.3
+
+If you are upgrading {mtc-full} ({mtc-short}) version 1.3.x, you must manually update the `indirectImageMigration` and `indirectVolumeMigration` parameters in the `MigPlan` custom resource (CR).
+
+Because the `indirectImageMigration` and `indirectVolumeMigration` parameters do not exist in version 1.3, their default value in version 1.4 is `false`, which means that direct image migration and direct volume migration are enabled. If the direct migration requirements are not fulfilled, the migration plan does not reach a `Ready` state.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/migration-upgrading-from-mtc-1-3.adoc:11]: Actually, they could do it, in theory. But it's unlikely that they would do so by the time of their first migration after the upgrade, so I will rewrite this section

File: modules/migration-upgrading-from-mtc-1-3.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-upgrading-from-mtc-1-3.adoc:50]: It's basically what is above:

spec:
  indirectVolumeMigration = true
  indirectImageMigration = true

The rest of the CR is unchanged.

**Pull request merged**